test/libyul/objectCompiler/eof/256_subcontainers.yul

```diff
@@ -0,0 +1,1570 @@
+//       stop
+// }
+//
+// sub_228: assembly {
+//         /* "source":11108:11115   */
+//       stop
+// }
+//
+// sub_229: assembly {
+//         /* "source":11156:11163   */
+//       stop
+// }
+//
+// sub_230: assembly {
+//         /* "source":11204:11211   */
+//       stop
+// }
+//
+// sub_231: assembly {
+//         /* "source":11252:11259   */
+//       stop
+// }
+//
+// sub_232: assembly {
+//         /* "source":11300:11307   */
+//       stop
+// }
+//
+// sub_233: assembly {
+//         /* "source":11348:11355   */
+//       stop
+// }
+//
+// sub_234: assembly {
+//         /* "source":11396:11403   */
+//       stop
+// }
+//
+// sub_235: assembly {
+//         /* "source":11444:11451   */
+//       stop
+// }
+//
+// sub_236: assembly {
+//         /* "source":11492:11499   */
+//       stop
+// }
+//
+// sub_237: assembly {
+//         /* "source":11540:11547   */
+//       stop
+// }
+//
+// sub_238: assembly {
+//         /* "source":11588:11595   */
+//       stop
+// }
+//
+// sub_239: assembly {
+//         /* "source":11636:11643   */
+//       stop
+// }
+//
+// sub_240: assembly {
+//         /* "source":11684:11691   */
+//       stop
+// }
+//
+// sub_241: assembly {
+//         /* "source":11732:11739   */
+//       stop
+// }
+//
+// sub_242: assembly {
+//         /* "source":11780:11787   */
+//       stop
+// }
+//
+// sub_243: assembly {
+//         /* "source":11828:11835   */
+//       stop
+// }
+//
+// sub_244: assembly {
+//         /* "source":11876:11883   */
+//       stop
+// }
+//
+// sub_245: assembly {
+//         /* "source":11924:11931   */
+//       stop
+// }
+//
+// sub_246: assembly {
+//         /* "source":11972:11979   */
+//       stop
+// }
+//
+// sub_247: assembly {
+//         /* "source":12020:12027   */
+//       stop
+// }
+//
+// sub_248: assembly {
+//         /* "source":12068:12075   */
+//       stop
+// }
+//
+// sub_249: assembly {
+//         /* "source":12116:12123   */
+//       stop
+// }
+//
+// sub_250: assembly {
+//         /* "source":12164:12171   */
+//       stop
+// }
+//
+// sub_251: assembly {
+//         /* "source":12212:12219   */
+//       stop
+// }
+//
+// sub_252: assembly {
+//         /* "source":12260:12267   */
+//       stop
+// }
+//
+// sub_253: assembly {
+//         /* "source":12308:12315   */
+//       stop
+// }
+//
+// sub_254: assembly {
+//         /* "source":12356:12363   */
+//       stop
+// }
+//
+// sub_255: assembly {
+//         /* "source":12404:12411   */
+//       stop
+// }
+// Bytecode: ef0001010004020001000c0300010014040000000080ffff5f808080ec005f5260205ff3ef00010100040200010001040000000080ffff00
+// Opcodes: 0xEF STOP ADD ADD STOP DIV MUL STOP ADD STOP 0xC SUB STOP ADD STOP EQ DIV STOP STOP STOP STOP DUP1 SELFDESTRUCT SELFDESTRUCT PUSH0 DUP1 DUP1 DUP1 EOFCREATE 0x0 PUSH0 MSTORE PUSH1 0x20 PUSH0 RETURN 0xEF STOP ADD ADD STOP DIV MUL STOP ADD STOP ADD DIV STOP STOP STOP STOP DUP1 SELFDESTRUCT SELFDESTRUCT STOP
```

Comment:
The bytecode is very short so the containers are being removed because they are unreferenced?

I guess this is enough to test the loop in `findReferencedContainers()`, but it would be better if they were all going all the way to the bytecode to make sure there's nothing else that will break at the limit. How about we do this?

``` yul
    object "s0000" { code{ pop(eofcreate("s0001", 0, 0, 0, 0)) } }
    object "s0001" { code{ pop(eofcreate("s0002", 0, 0, 0, 0)) } }
    object "s0002" { code{ pop(eofcreate("s0003", 0, 0, 0, 0)) } }
    ...
```

Reply:
This will not work. `eofcreate` can ba called only with a subcontainer, but I will just add this references in the main container and create all 256 contracts there.

Reply:
Right, forgot about that detail. The way you did it is good.

The test ended up being huge, but I guess that's unavoidable when we're testing the limits of the compiler.